azkaban-web-server/src/main/java/azkaban/webapp/WebMetrics.java

```diff
@@ -38,8 +37,13 @@
   // How long does user log fetch take when user call fetch-log api.
   private final AtomicLong logFetchLatency = new AtomicLong(0L);
 
-  WebMetrics() {
-    this.registry = MetricsManager.INSTANCE.getRegistry();
+  @Inject
+  WebMetrics(final MetricRegistry registry) {
+    // Add unit test coverage to guarantee no duplication
+    // Seeing this error after guicifying, but before binding as singleton in AzkabanWebServerModule:
```

Comment:
Yes, the problem has already been fixed. I just included this because there was no unit test confirming that WebMetrics is a singleton, which allowed me to build and run and then find the bug while using my local version. Adding `bind(WebMetrics.class).in(Scopes.SINGLETON);` to AzkabanWebServerModule fixed the problem, but it would probably be good to have a unit test in case anybody removes it for whatever reason. I didn't think it'd be wise to include it in this (already very large) PR, so I added the comment. What do you think?